recipe/meta.yaml

```
@@ -1,26 +1,26 @@
 {% set name = "hvsampledata" %}
-{% set version = "0.1.1" %}
+{% set version = "0.1.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/hvsampledata-{{ version }}.tar.gz
-  sha256: 81d6f159beddbd01060546740aba1df946cda24ce4dcce31339103a3c287a190
+  sha256: 9f950bd1fbc74384c14c00d4e9dfa0b08956e0db9f19de5031caf999d012b11c
 
 build:
   number: 0
   # panel >=0.13.0 and nodejs aren't available on s390x
-  skip: True # [py<39 or s390x]
+  skip: true  # [py<39 or s390x]
```

```suggestion
  skip: true  # [py<39]
```
It's not needed anymore as we don't build on that platform.

---

True! The bot could catch that fix that I guess.

---

@msentissi For your awareness. I can make a ticket to track. This should be possible with the `SelectorParser` class, but I think this work is probably best suited as an AnacondaLinter rule.

---

@cbouss for visibility/confirmation of including this as a linter auto-fix rule

---

Tracking here: https://anaconda.atlassian.net/browse/PI-227

---

Noted, thank you!